e2e_tests/conftest.py

```diff
@@ -127,6 +127,7 @@ def create_aau_keycloak_user(keycloak_admin, create_keycloak_user):
 def create_standard_keycloak_user(keycloak_admin, create_keycloak_user):
     user_groups = [
         "/ayr_user_type/view_dept",
+        "/ayr_user_type/download",
```

Comment:
I think its worth us adding  end to end test for a happy path and unhappy path for the button and download route to show that we have configured thiings correctly e2e, since the other tests mock keycloak groups